### DISCUSSION TITLE
似乎不支持require的导入方式，如果打包成exe，会出问题

### DISCUSSION THREAD

**Discussion by w-cyh** [wechatferry/wechatferry#5] [General]
似乎不支持require的导入方式，如果打包成exe，会出问题

**Comment by KeJunMao** [comment]
这个是纯 esm 包，请自己使用 bundle 工具转哈

**Reply by w-cyh** [reply]
尝试了，转失败了

**Reply by KeJunMao** [reply]
那就直接用 esm

**Reply by KeJunMao** [reply]
可以看下 example 文件夹

**Comment by w-cyh** [comment]
![image](https://github.com/user-attachments/assets/9a5f0e21-08b8-42a9-aea0-8ca07ef8292b)
在vue和electron的项目中通过import的引入，控制台会报这个提示，但是不影响使用

**Reply by KeJunMao** [reply]
PR Welcome

**Reply by w-cyh** [reply]
![image](https://github.com/user-attachments/assets/16c532a6-fc54-4f8a-8c52-3171739c4f81)
是不能在这里引用吗打包也报错
